cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/epilog

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+exec &>>"/var/log/parallelcluster/slurm_prolog_epilog.log"
+timestamp=`date +"%Y-%m-%d %H:%M:%S,%3N"`
+echo "$timestamp Removing nodes belonging to job ${SLURM_JOB_ID} from /etc/hosts"
+sed -i "/#HOSTS_JOB_${SLURM_JOB_ID}/,/#END_JOB_${SLURM_JOB_ID}/d" /etc/hosts
```

Comment:
can we handle errors for sed execution and print the proper logging message? we might want to include the output of the sed command when it fails in the logs.

Reply:
We could because the stdout and stderr were all redirected.

With my new simplified code, the errors are also handled properly